drive/types.bal

```diff
@@ -460,15 +485,23 @@ public type CreateFileOptional record {
 # + includePermissionsForView - Specifies which additional view's permissions to include in the response. 
 #                               Only 'published' is supported.
 # + addParents - A comma-separated list of parent IDs to add.
-# + supportsAllDrives -   
+# + supportsAllDrives -  Whether the requesting application supports both My Drives and shared drives. (Default: false)  
+@display {label: "File Metadata Optional Parameters"}
```

Comment:
There is no need to say `Optional Parameters`. It is acquired by analysing syntax tree. File Metadata is enough?

Reply:
Yeah. We will update like that. Here is a record for `optional query paramters` in file update

Reply:
Updated with the new label